### PR TITLE
Add WordCounter v1.2.1

### PR DESCRIPTION
**Pull request by mence**
This adds support for installing the [WordCounter](http://wordcounterapp.com/) app.

### CODE REVIEW

File: Casks/wordcounter.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'wordcounter' do
+  version '1.2.1'
+  sha256 'fdbda9b84346462f1c10b99f8a987b55cce681e21d8f07de22560286d671bd0e'
+
+  url 'http://wordcounterapp.com/download'
+  name 'WordCounter'
+  homepage 'http://wordcounterapp.com/'
+  license :unknown
```
Comment (vitorgalvao) [inline Casks/wordcounter.rb]: `:commercial`.

**Comment by vitorgalvao** [comment]
Seeing as `url` doesn’t point to a specific version, this cask should use `version :latest` and `sha256 :no_check`. [See an example](https://github.com/caskroom/homebrew-cask/blob/master/Casks/acorn.rb).

**Comment by mence** [comment]
@vitorgalvao thanks for the help, it's my first Cask, so wasn't entirely sure of some of those fields. I've updated the PR with your suggestions.

**Comment by vitorgalvao** [comment]
Looks great.

**Pull request merged**